src/main/resources/liquibase/crop_changelog/24_3_0.xml

```diff
@@ -0,0 +1,390 @@
+			
+			set @cvId = (select max(cv_id) from cv where definition = "Packing_Status");
+			
+			INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (@cvId, '1', '1=packed', null, 0, 0);
+			
+			set @catvalue=(select max(cvterm_id) from cvterm);
+			
+			INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1190, @scaleId, @catvalue);
+
+			INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (@cvId, 2, '2=skipped', null, 0, 0);
+			set @catvalue=(select max(cvterm_id) from cvterm);
+			
+			INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1190, @scaleId, @catvalue);
+
+        </sql>
+    </changeSet>	
+	
+	 <changeSet id="v24.3.0-8" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Plot Number" AND cv_id = 1010;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add cvterm Plot Number </comment>
+        <sql dbms="mysql" splitStatements="true">
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1010, 'Plot Number', 'Plot Number', null, 0, 0);
+        </sql>
+    </changeSet>				
+				
+    <changeSet id="v24.3.0-9" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Assigned Passport Descriptor" AND cv_id = 1020;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add cvterm entry for Method Assigned Passport Descriptor</comment>
+        <sql dbms="mysql" splitStatements="true">
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1020, 'Assigned Passport Descriptor', 'Standard method for assigned passport descriptor', null, 0, 0);
+        </sql>
+    </changeSet>
+
+    <changeSet id="v24.3.0-10" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Text" AND cv_id = 1030;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add cvterm entry for Scale Text</comment>
+        <sql dbms="mysql" splitStatements="true">
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1030, 'Text', 'Text', null, 0, 0);
+        </sql>
+    </changeSet>   
+	
+	<changeSet id="v24.3.0-11" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "PLOT_NUMBER_AP_text" and cv_id = 1040 ;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add variable for Packing _Status</comment>
```

Comment:
Hi @rvtulod is this comment correct? Seems like a new variable is created PLOT_NUMBER_AP_text